packages/core/test/snapshots/policySchema.spec.js.md

```diff
@@ -0,0 +1,69 @@
+# Snapshot report for `test/policySchema.spec.js`
+
+The actual snapshot is saved in `policySchema.spec.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## policy schema - invalid policy - empty
+
+> Snapshot 1
+
+    [
+      {
+        instancePath: '',
+        keyword: 'required',
+        message: 'must have required property \'resources\'',
+        params: {
+          missingProperty: 'resources',
+        },
+        schemaPath: '#/anyOf/0/required',
+      },
+      {
+        instancePath: '',
+        keyword: 'required',
+        message: 'must have required property \'resolutions\'',
+        params: {
+          missingProperty: 'resolutions',
+        },
+        schemaPath: '#/anyOf/1/required',
+      },
+      {
+        instancePath: '',
+        keyword: 'anyOf',
+        message: 'must match a schema in anyOf',
+        params: {},
+        schemaPath: '#/anyOf',
+      },
+    ]
+
+## policy schema - invalid policy - bad pkg name
+
+> Snapshot 1
+
+    [
+      {
+        instancePath: '/resources',
+        keyword: 'additionalProperties',
+        message: 'must NOT have additional properties',
```

Comment:
This is not great. I know JSON schema can't do better. Let's skip validating resource IDs altogether.